server/package.json

```diff
@@ -85,6 +85,7 @@
     "graphql-yoga": "^4.0.4",
     "jest-mock-extended": "^3.0.4",
     "jsonwebtoken": "^9.0.0",
+    "lodash": "^4.17.21",
```

Comment:
we don't load the full loadash, only the functions that we need! (see examples below)